exporter/elasticsearchexporter/bulkindexer.go

```diff
@@ -50,27 +50,34 @@ type bulkIndexerSession interface {
 	Flush(context.Context) error
 }
 
+const defaultMaxRetries = 2
```

Comment:
[to reviewer] moving default out of default config, otherwise we cannot detect max_requests !=0 and max_retries != 0 due to default config being initialized first.